201-redis-premium-vnet/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Create Premium Redis Cache with Virtual Network",
+  "description": "This template shows how to configure both Virtual Network support for a premium Azure Redis Cache instance",
+  "summary": "This template provides the benefit of Virtual Network support.",
```

Comment:
This template provides a Redis Cache with the benefit of Virtual Network support.